### PR TITLE
[WebApp] Add logic of web_app form to telegram 

### PR DESCRIPTION
**Pull request by Teri-anric**
- Made the `ip_address` field in the `NotificationData` model nullable.
- Added a new migration to reflect the change in the database schema.
- Introduced a new notification creation feature in the Telegram bot, allowing users to create subscriptions with wallet ID, health ratio level, and protocol selection.

### CODE REVIEW

File: apps/web_app/telegram/crud.py [hunk 1]
```diff
@@ -108,3 +108,8 @@ async def get_objects_by_filter(
             if limit == 1:
                 return await db.scalar(stmp)
             return await db.scalars(stmp).all()
+
+    async def write_to_db(self, obj: ModelType) -> None:
+        async with self.Session() as db:
```
Comment (djeck1432) [inline apps/web_app/telegram/crud.py:113]: add docstring please

File: apps/web_app/telegram/handlers/create_notification.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+                [types.InlineKeyboardButton(text="Cancel", callback_data="cancel_form")]
+            ]
+        ),
+    )
+
+
+@create_notification_router.message(NotificationFormStates.health_ratio_level)
+async def process_health_ratio(message: types.Message, state: FSMContext):
+    """Processes the health ratio level input from the user."""
+    try:
```
Comment (djeck1432) [inline apps/web_app/telegram/handlers/create_notification.py:51]: use try except only to expected lines which can failed `float(message.text)` can be just covered with try/except other code can be moved out

File: apps/web_app/telegram/handlers/create_notification.py [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+        health_ratio = float(message.text)
+        if not (0 <= health_ratio <= 10):
```
Comment (djeck1432) [inline apps/web_app/telegram/handlers/create_notification.py:53]: move `0` and `10` to constant, it's called magic number

File: apps/web_app/telegram/handlers/create_notification.py [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+        for protocol in ProtocolIDs:
+            protocol_buttons.append(
+                [
+                    types.InlineKeyboardButton(
+                        text=protocol.value, callback_data=f"protocol_{protocol.value}"
+                    )
+                ]
+            )
+        protocol_buttons.append(
+            [types.InlineKeyboardButton(text="Cancel", callback_data="cancel_form")]
```
Comment (djeck1432) [inline apps/web_app/telegram/handlers/create_notification.py:70]: why not move it there, where you init `protocol_buttons` variable?

**Review by djeck1432 — CHANGES_REQUESTED** [review]

File: apps/web_app/telegram/handlers/create_notification.py (hunk 4 above)
Comment (Teri-anric) [reply apps/web_app/telegram/handlers/create_notification.py]: I wanted it to be from bottom.
fixed by moving to kb and usage keyboard builder

**Pull request merged**